networking.md

```diff
@@ -506,4 +506,166 @@ spec:
+```
+
+The `IPSet` CRD can be used by a requestor to register an IP address.
+
+The request is a list of networks with `name` and `subnetName`, optional `defaultRoute` for one subnet in total. Using the `fixedIP` a specific IP can be requested, if not already assigned. The `fixedIP` don't have to be from within an `allocationRange`.
+
+The status if the `IPSet` will have all details about the reserved IPs and network details to be used by the requestor. The `IPSet` can be used by the request if the overall `Ready` condition is `True`.
+
+Example to request IP reservation with requesting a `fixedIP` on the `InternalApi` network `subnet1`:
+```yaml
+kind: IPSet
+metadata:
+  annotations:
+    kubectl.kubernetes.io/last-applied-configuration: |
+      {"apiVersion":"network.openstack.org/v1beta1","kind":"IPSet","metadata":{"annotations":{},"name":"edpm-compute-0","namespace":"openstack"},"spec":{"networks":[{"defaultRoute":true,"name":"CtlPlane","subnetName":"subnet1"},{"fixedIP":"172.17.0.150","name":"InternalApi","subnetName":"subnet1"},{"name":"Storage","subnetName":"subnet1"},{"name":"Tenant","subnetName":"subnet1"},{"name":"External","subnetName":"subnet1"}]}}
+  creationTimestamp: "2023-05-25T07:27:31Z"
+  finalizers:
+  - IPSet
+  generation: 1
+  name: edpm-compute-0
+  namespace: openstack
+  resourceVersion: "4916903"
+  uid: 3901b2f3-f724-48fb-a906-7b46e9a5b19a
+spec:
+  networks:
+  - defaultRoute: true
+    name: CtlPlane
+    subnetName: subnet1
+  - fixedIP: 172.17.0.150
+    name: InternalApi
+    subnetName: subnet1
+  - name: Storage
+    subnetName: subnet1
+  - name: Tenant
+    subnetName: subnet1
+  - name: External
+    subnetName: subnet1
+status:
+  conditions:
+  - lastTransitionTime: "2023-05-25T07:27:31Z"
+    message: Setup complete
+    reason: Ready
+    status: "True"
+    type: Ready
+  - lastTransitionTime: "2023-05-25T07:27:31Z"
+    message: Input data complete
+    reason: Ready
+    status: "True"
+    type: InputReady
+  - lastTransitionTime: "2023-05-25T07:27:31Z"
+    message: Reservation successful
+    reason: Ready
+    status: "True"
+    type: ReservationReady
+  reservations:
+  - address: 192.168.122.100
+    cidr: 192.168.122.0/24
+    gateway: 192.168.122.1
+    mtu: 1500
+    network: CtlPlane
+    subnet: subnet1
+  - address: 10.0.0.10
+    cidr: 10.0.0.0/24
+    gateway: 10.0.0.1
+    mtu: 1500
+    network: External
+    routes:
+    - destination: 10.0.1.0/24
+      nexthop: 10.0.0.100
+    subnet: subnet1
+  - address: 172.17.0.150
+    cidr: 172.17.0.0/24
+    mtu: 1500
+    network: InternalApi
+    subnet: subnet1
+    vlan: 20
+  - address: 172.18.0.10
+    cidr: 172.18.0.0/24
+    mtu: 1500
+    network: Storage
+    subnet: subnet1
+    vlan: 30
+  - address: 172.20.0.10
+    cidr: 172.20.0.0/24
+    mtu: 1500
+    network: Tenant
+    subnet: subnet1
+    vlan: 50
+```
+
+The reservation is also storted via the `Reservation` CRD. The CRD is meant to be "internal" and not direct used.
```

Comment:
nit: s/storted/stored/

I think using comma nd refering to `Reservation` CRD as "it" would be better
 s/... `Reservation` CRD. The CRD is meant to be "internal" and not direct used./... `Reservation` CRD, but it is intended for "internal" use only.